utils/mc/Makefile

```
@@ -125,4 +125,19 @@ define Package/mc/conffiles
 /etc/mc/skins/default.ini
 endef
 
+define Package/mc-skins
+	TITLE:=Midnight Commander - a powerful visual file manager - skins
+	URL:=https://www.midnight-commander.org/
+	SECTION:=utils
+	CATEGORY:=Utilities
+	DEPENDS:=+mc
```

```suggestion
  TITLE:=Midnight Commander - a powerful visual file manager - skins
  URL:=https://www.midnight-commander.org/
  SECTION:=utils
  CATEGORY:=Utilities
  DEPENDS:=+mc
```

I think the **TITLE** might be so long that it does not display in `make menuconfig`, it needs to be verified.

---

> I think the TITLE might be so long that it does not display in `make menuconfig`, it needs to be verified.

It fits for me.

---

Ok, thanks for testing!